src/pollyCLI/pollyintegration.cpp

```diff
@@ -302,18 +302,34 @@ QVariantMap PollyIntegration::getUserProjectFiles(QStringList ProjectIds){
 // PURPOSE:
 //    This function parses the output of "createproject" command run from Qtprocess..
 // Return project id for the new project..
-// CALLS TO: run_qt_process,get_run_id
+// CALLS TO: run_qt_process,parse_id
 //
 // CALLED FROM: external clients
 
 
 QString PollyIntegration::createProjectOnPolly(QString projectname){
     QString command2 = "createProject";
     QList<QByteArray> result_and_error = run_qt_process(command2, QStringList() << credFile<< projectname);
-    QString run_id = get_run_id(result_and_error.at(0));
+    QString run_id = parse_id(result_and_error.at(0));
     return run_id;
 }
 
+// name OF FUNCTION: createWorkflowRequest
+// PURPOSE:
+//    This function parses the output of "createWorkflowRequest" command run from Qtprocess..
```

Comment:
@Ky5595 Please use the doxygen format for function documentation. The details are available in the Contributing guidelines.
https://github.com/ElucidataInc/ElMaven/blob/develop/CONTRIBUTING.md